src/Presentation/ConfigEditPageTextBuilder.php

```diff
@@ -43,7 +43,6 @@ public function createBottomHtml(): string {
 				'msg-wikibase-entity-item' => $this->context->msg( 'wikibase-entity-item' )->plain(),
 				'msg-wikibase-faceted-search-config-tab-name' => $this->context->msg( 'wikibase-faceted-search-config-tab-name' )->plain(),
 				'msg-wikibase-faceted-search-config-tab-name-missing' => $this->context->msg( 'wikibase-faceted-search-config-tab-name-missing' )->plain(),
-				'msg-wikibase-faceted-search-config-help' => $this->context->msg( 'wikibase-faceted-search-config-help' )->escaped(),
```

Comment:
What I just re-learned: the `plain` (rather than `escaped`) calls for the other messages are fine, because Mustache by-default escapes variables